clang/test/dpct/asm/lop3.cu

```diff
@@ -7,32 +7,45 @@
 // a^b^c
 static __device__ __forceinline__ uint32_t LOP3LUT_XOR(uint32_t a, uint32_t b, uint32_t c) {
   uint32_t d1;
-  // CHECK: d1 = a ^ b ^ c;
+  // CHECK: d1 = dpct::lop3(a, b, c, 0x96);
   asm("lop3.b32 %0, %1, %2, %3, 0x96;" : "=r"(d1) : "r"(a), "r"(b), "r"(c));
   return d1;
 }
 
 // (a ^ (c & (b ^ a)))
 static __device__ __forceinline__ uint32_t LOP3LUT_XORAND(uint32_t a, uint32_t b, uint32_t c) {
   uint32_t d2;
-  // CHECK: d2 = (a ^ (c & (b ^ a)));
+  // CHECK: d2 = dpct::lop3(a, c, b, 0xb8);
```

Comment:
for known lut value, prefer to keep not to use the helper function.

Reply:
okay, will update in the next commit.